pages/common/bioradtopgm.md

```diff
@@ -0,0 +1,12 @@
+# bioradtopgm
+
+> Convert a Biorad confocal file into a PGM file.
+> More information: <https://netpbm.sourceforge.net/doc/bioradtopgm.html>.
+
+- Read a Biorad confocal file as input and produce a PGM image as output:
+
+`bioradtopgm {{path/to/input_file}} > {{path/to/output_file.pgm}}`
+
```

Comment:
Could you please also add an example that demonstrates the use of the program without the `-image` option? Thanks!

Reply:
I'm not able to find an example in the sourceforge documentation, What would you suggest @gutjuri?

Reply:
Unfortunately I have no biorad confocal image file, so I can't really try this out. But from reading the docs, my interpretation of the command is as follows:
- If you specify `-image {{n}}`, then the n'th image in the input file is converted
- If you don't specify `-image`, then no images are converted but info about the input file is printed
- If you specify `-image` but leave out the `{{n}}`, then there's an error

However, there is oine different interpretation of the docs, whoich could also be valid:
- If you specify `-image {{n}}`, then the n'th image in the input file is converted
- If you don't specify `-image`, then all/the first image(s) in the input file are converted
- If you specify `-image` but leave out the `{{n}}`, then no images are converted but info about the input file is printed

So, I'm not sure how to proceed here. As BioRad image files are probably only available to specialists, I don't see any way to find out what is correct without reading the source code. I might do this later, so be patient please :D

Reply:
No problem! :) 
Thanks for the insights. Meanwhile, I'll do my research and commit if I find something resourceful